### ISSUE TITLE
Error message on startup using nvim latest stable (ubuntu 20.04)

### ISSUE THREAD

**Issue by arooni** [tools-life/taskwiki#377]
ENV:  Ubuntu 20.04 

```
925d8ed [7 days ago] (HEAD -> master, origin/master, origin/HEAD) README: Adjust old tasklib install instructions [Tomas Babej]

[I]  ~/.v/p/taskwiki   master   python3 --version
Python 3.8.10
[I]  ✘  ~/.v/p/taskwiki   master   task --version
2.6.1
[I]  ~/.v/p/taskwiki   master  
[I]  ~/.v/p/taskwiki   master   nvim --version
NVIM v0.5.1
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/david/software/neovim/build/config -I/home/david/software/neovim/src -I/home/david/software/neovim/.deps/usr/include -I/usr/include -I/home/david/software/neovim/build/src/nvim/auto -I/home/david/software/neovim/build/include
Compiled by david@LilArooni

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/share/nvim"

Run :checkhealth for more info
```

error message is:

```
Error detected while processing function provider#python3#Call:
line   18:
Error invoking 'python_execute' on channel 3 (python3-script-host):
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/david/.vim/plugged/taskwiki/taskwiki/errors.py", line 26, in wrapped_function
    original_function(*args, **kwargs)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/decorators.py", line 10, in wrapped_function
    original_function(*args, **kwargs)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/main.py", line 41, in update_from_tw
    c.load_vwtasks(buffer_has_authority=False)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/cache.py", line 181, in load_vwtasks
    self.vwtask[i]  # Loads the line into the cache
  File "/home/david/.vim/plugged/taskwiki/taskwiki/store.py", line 58, in __getitem__
    item = self.get_method(key)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/store.py", line 128, in get_method
    return vwtask.VimwikiTask.from_line(self.cache, line)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/vwtask.py", line 103, in from_line
    if cache.buffer_has_authority or not self.task.saved:
  File "/home/david/.vim/plugged/taskwiki/taskwiki/vwtask.py", line 211, in task
    return self.cache.task[self.uuid]
  File "/home/david/.vim/plugged/taskwiki/taskwiki/store.py", line 58, in __getitem__
    item = self.get_method(key)
  File "/home/david/.vim/plugged/taskwiki/taskwiki/store.py", line 107, in get_method
    return key.tw.tasks.get(uuid=key.value)
  File "/usr/local/lib/python3.8/dist-packages/tasklib/task.py", line 548, in get
    num = len(clone)
  File "/usr/local/lib/python3.8/dist-packages/tasklib/task.py", line 471, in __len__
    self._result_cache = list(self.__iter__())
  File "/usr/local/lib/python3.8/dist-packages/tasklib/task.py", line 476, in __iter__
    self._result_cache = self._execute()
  File "/usr/local/lib/python3.8/dist-packages/tasklib/task.py", line 508, in _execute
    return self.backend.filter_tasks(self.filter_obj)
  File "/usr/local/lib/python3.8/dist-packages/tasklib/backends.py", line 334, in filter_tasks
    for line in self.execute_command(args):  File "/usr/local/lib/python3.8/dist-packages/tasklib/backends.py", line 297, in execute_command
    raise TaskWarriorException(error_msg)
tasklib.backends.TaskWarriorException: TASKRC override: /home/david/.taskrc
Configuration override rc.confirmation=no
Configuration override rc.dependency.confirmation=no
Configuration override rc.recurrence.confirmation=no
Configuration override rc.json.array=off
Configuration override rc.bulk=0
Configuration override rc.context=
Unable to find report that matches '87c5c563'.
Command used: task rc.confirmation=no rc.dependency.confirmation=no rc.recurrence.confirmation=no rc.json.array=off rc.bulk=0 rc.context= export 87c5c563


```

**Comment by arooni** [comment]
In addition, there are times I can't even get vimwwiki to launch

nvim --startup-time nvim.log ~/Documents/vimwiki/index.wiki 

yields the following, and nothing seems to happen after about time + 600 ms

```


times in msec
 clock   self+sourced   self:  sourced script
 clock   elapsed:              other lines

000.017  000.017: --- NVIM STARTING ---
000.880  000.863: locale set
001.566  000.685: inits 1
001.596  000.031: window checked
001.691  000.094: parsing arguments
001.906  000.216: expanding arguments
001.992  000.086: inits 2
003.181  001.189: init highlight
003.190  000.008: waiting for UI
004.471  001.281: done waiting for UI
004.510  000.040: init screen for UI
006.135  000.761  000.761: sourcing /usr/local/share/nvim/runtime/syntax/syncolor.vim
007.157  001.977  001.216: sourcing /usr/local/share/nvim/runtime/syntax/synload.vim
007.314  000.064  000.064: sourcing /home/david/.config/nvim/filetype.vim
025.626  018.198  018.198: sourcing /usr/local/share/nvim/runtime/filetype.vim
028.824  000.912  000.912: sourcing /usr/local/share/nvim/runtime/scripts.vim
029.347  024.375  003.224: sourcing /usr/local/share/nvim/runtime/syntax/syntax.vim
030.265  000.260  000.260: sourcing /usr/local/share/nvim/runtime/syntax/syncolor.vim
031.640  000.019  000.019: sourcing /home/david/.config/nvim/filetype.vim
031.728  000.017  000.017: sourcing /usr/local/share/nvim/runtime/filetype.vim
032.208  000.060  000.060: sourcing /usr/local/share/nvim/runtime/ftplugin.vim
032.689  000.053  000.053: sourcing /usr/local/share/nvim/runtime/indent.vim
041.963  000.028  000.028: sourcing /home/david/.config/nvim/filetype.vim
042.110  000.030  000.030: sourcing /usr/local/share/nvim/runtime/filetype.vim
042.897  000.031  000.031: sourcing /usr/local/share/nvim/runtime/ftplugin.vim
043.518  000.016  000.016: sourcing /usr/local/share/nvim/runtime/macros/matchit.vim
050.080  003.216  003.216: sourcing /home/david/.config/nvim/autoload/plug.vim
065.586  001.285  001.285: sourcing /usr/local/share/nvim/runtime/ftoff.vim
074.964  000.050  000.050: sourcing /home/david/.config/nvim/filetype.vim
084.367  000.155  000.155: sourcing /home/david/.vim/plugged/vim-ruby/ftdetect/ruby.vim
084.658  000.261  000.261: sourcing /home/david/.vim/plugged/vim-ruby/ftdetect/ruby_extra.vim
085.014  000.022  000.022: sourcing /home/david/.vim/plugged/vim-fugitive/ftdetect/fugitive.vim
085.121  000.020  000.020: sourcing /home/david/.vim/plugged/vim-javascript/ftdetect/flow.vim
085.199  000.054  000.054: sourcing /home/david/.vim/plugged/vim-javascript/ftdetect/javascript.vim
085.468  000.034  000.034: sourcing /home/david/.vim/plugged/ultisnips/ftdetect/snippets.vim
085.705  000.073  000.073: sourcing /home/david/.vim/plugged/vim-markdown/ftdetect/markdown.vim
087.635  012.282  011.664: sourcing /usr/local/share/nvim/runtime/filetype.vim
088.473  000.021  000.021: sourcing /usr/local/share/nvim/runtime/ftplugin.vim
089.247  000.018  000.018: sourcing /usr/local/share/nvim/runtime/indent.vim
091.419  000.274  000.274: sourcing /usr/local/share/nvim/runtime/syntax/syncolor.vim
092.817  000.288  000.288: sourcing /usr/local/share/nvim/runtime/syntax/syncolor.vim
097.252  007.574  007.012: sourcing /home/david/.config/nvim/colors/solarized.vim
097.712  092.835  043.499: sourcing /home/david/.config/nvim/init.vim
097.720  000.375: sourcing vimrc file(s)
099.990  001.794  001.794: sourcing /home/david/.vim/plugged/bufexplorer/plugin/bufexplorer.vim
101.339  001.242  001.242: sourcing /home/david/.vim/plugged/vim-rails/plugin/rails.vim
101.707  000.181  000.181: sourcing /home/david/.vim/plugged/vim-signature/autoload/signature/utils.vim
103.251  001.816  001.634: sourcing /home/david/.vim/plugged/vim-signature/plugin/signature.vim
103.832  000.233  000.233: sourcing /home/david/.vim/plugged/nerdtree/autoload/nerdtree.vim
105.002  000.674  000.674: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/path.vim
105.261  000.195  000.195: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/menu_controller.vim
105.440  000.101  000.101: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/menu_item.vim
105.639  000.136  000.136: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/key_map.vim
105.989  000.290  000.290: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/bookmark.vim
106.398  000.285  000.285: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/tree_file_node.vim
107.020  000.558  000.558: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/tree_dir_node.vim
107.326  000.245  000.245: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/opener.vim
107.694  000.309  000.309: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/creator.vim
107.819  000.065  000.065: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/flag_set.vim
108.062  000.182  000.182: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/nerdtree.vim
108.533  000.413  000.413: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/ui.vim
108.620  000.028  000.028: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/event.vim
108.782  000.101  000.101: sourcing /home/david/.vim/plugged/nerdtree/lib/nerdtree/notifier.vim
109.482  000.632  000.632: sourcing /home/david/.vim/plugged/nerdtree/autoload/nerdtree/ui_glue.vim
114.266  000.136  000.136: sourcing /home/david/.config/nvim/nerdtree_plugin/exec_menuitem.vim
114.744  000.452  000.452: sourcing /home/david/.config/nvim/nerdtree_plugin/fs_menu.vim
114.954  000.023  000.023: sourcing /home/david/.vim/plugged/nerdtree/nerdtree_plugin/exec_menuitem.vim
114.997  000.020  000.020: sourcing /home/david/.vim/plugged/nerdtree/nerdtree_plugin/fs_menu.vim
115.079  000.062  000.062: sourcing /home/david/.vim/plugged/nerdtree/nerdtree_plugin/vcs.vim
115.374  000.120  000.120: sourcing /home/david/.vim/plugged/nerdtree-git-plugin/autoload/gitstatus.vim
115.994  000.081  000.081: sourcing /home/david/.vim/plugged/nerdtree-git-plugin/autoload/gitstatus/log.vim
117.066  000.121  000.121: sourcing /home/david/.vim/plugged/nerdtree-git-plugin/autoload/gitstatus/listener.vim
117.279  002.114  001.792: sourcing /home/david/.vim/plugged/nerdtree-git-plugin/nerdtree_plugin/git_status.vim
118.832  015.483  008.230: sourcing /home/david/.vim/plugged/nerdtree/plugin/NERD_tree.vim
118.968  000.027  000.027: sourcing /home/david/.vim/plugged/vim-do/plugin/do.vim
122.096  003.051  003.051: sourcing /home/david/.vim/plugged/vim-dispatch/plugin/dispatch.vim
122.500  000.232  000.232: sourcing /home/david/.vim/plugged/vim-easy-align/plugin/easy_align.vim
122.779  000.179  000.179: sourcing /home/david/.vim/plugged/conoline.vim/plugin/conoline.vim
123.157  000.274  000.274: sourcing /home/david/.vim/plugged/emmet-vim/plugin/emmet.vim
124.645  001.395  001.395: sourcing /home/david/.vim/plugged/supertab/plugin/supertab.vim
125.630  000.210  000.210: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/init.vim
126.535  000.314  000.314: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/parts.vim
126.992  000.068  000.068: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/formatter/short_path.vim
129.569  000.544  000.544: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/util.vim
129.706  004.942  003.806: sourcing /home/david/.vim/plugged/vim-airline/plugin/airline.vim
129.875  000.061  000.061: sourcing /home/david/.vim/plugged/vim-airline-themes/plugin/airline-themes.vim
130.215  000.063  000.063: sourcing /home/david/.vim/plugged/tcomment_vim/autoload/tcomment/deprecated.vim
135.349  005.386  005.323: sourcing /home/david/.vim/plugged/tcomment_vim/plugin/tcomment.vim
136.260  000.749  000.749: sourcing /home/david/.vim/plugged/CamelCaseMotion/plugin/camelcasemotion.vim
137.425  001.046  001.046: sourcing /home/david/.vim/plugged/vim-surround/plugin/surround.vim
137.908  000.342  000.342: sourcing /home/david/.vim/plugged/vim-indent-object/plugin/indent-object.vim
141.530  003.513  003.513: sourcing /home/david/.vim/plugged/vim-fugitive/plugin/fugitive.vim
144.878  002.132  002.132: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/vars.vim
152.361  000.716  000.716: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/u.vim
163.204  000.110  000.110: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/markdown_base.vim
164.661  004.943  004.833: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/base.vim
165.150  000.243  000.243: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/path.vim
169.672  027.935  019.901: sourcing /home/david/.vim/plugged/vimwiki/plugin/vimwiki.vim
170.102  000.037  000.037: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/autoloclist.vim
170.167  000.032  000.032: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/balloons.vim
170.216  000.018  000.018: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/checker.vim
170.265  000.019  000.019: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/cursor.vim
170.318  000.020  000.020: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/highlighting.vim
170.365  000.018  000.018: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/loclist.vim
170.414  000.018  000.018: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/modemap.vim
170.462  000.018  000.018: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/notifiers.vim
170.510  000.018  000.018: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/registry.vim
170.558  000.019  000.019: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/signs.vim
171.528  000.629  000.629: sourcing /home/david/.vim/plugged/syntastic/autoload/syntastic/util.vim
176.980  000.081  000.081: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/autoloclist.vim
177.091  000.079  000.079: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/balloons.vim
177.382  000.261  000.261: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/checker.vim
177.536  000.125  000.125: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/cursor.vim
177.674  000.109  000.109: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/highlighting.vim
178.104  000.402  000.402: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/loclist.vim
178.253  000.116  000.116: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/modemap.vim
178.384  000.094  000.094: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/notifiers.vim
179.795  001.380  001.380: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/registry.vim
179.968  000.134  000.134: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic/signs.vim
182.306  011.720  008.310: sourcing /home/david/.vim/plugged/syntastic/plugin/syntastic.vim
183.059  000.607  000.607: sourcing /home/david/.vim/plugged/vim-pencil/plugin/pencil.vim
183.495  000.299  000.299: sourcing /home/david/.vim/plugged/undotree/plugin/undotree.vim
184.987  000.705  000.705: sourcing /home/david/.vim/plugged/dash.vim/autoload/dash/defaults.vim
185.194  001.166  000.462: sourcing /home/david/.vim/plugged/dash.vim/autoload/dash.vim
186.039  002.429  001.262: sourcing /home/david/.vim/plugged/dash.vim/plugin/dash.vim
187.485  000.163  000.163: sourcing /usr/local/share/nvim/runtime/autoload/provider/pythonx.vim
188.450  000.370  000.370: sourcing /usr/local/share/nvim/runtime/autoload/remote/host.vim
188.644  001.873  001.340: sourcing /usr/local/share/nvim/runtime/autoload/provider/python3.vim
188.756  002.581  000.708: sourcing /home/david/.vim/plugged/MatchTagAlways/plugin/MatchTagAlways.vim
190.081  000.503  000.503: sourcing /home/david/.vim/plugged/delimitMate/autoload/delimitMate.vim
193.925  005.050  004.547: sourcing /home/david/.vim/plugged/delimitMate/plugin/delimitMate.vim
194.659  000.598  000.598: sourcing /home/david/.vim/plugged/vim-closetag/plugin/closetag.vim
195.793  001.023  001.023: sourcing /home/david/.fzf/plugin/fzf.vim
195.944  000.032  000.032: sourcing /home/david/.vim/plugged/vim-tips/plugin/vim-tips.vim
196.131  000.095  000.095: sourcing /home/david/.vim/plugged/calendar.vim/plugin/calendar.vim
196.514  000.292  000.292: sourcing /home/david/.vim/plugged/vim-auto-save/plugin/AutoSave.vim
232.427  035.800  035.800: sourcing /home/david/.vim/plugged/vim-taskwarrior/plugin/taskwarrior.vim
234.621  000.936  000.936: sourcing /home/david/.vim/plugged/unite.vim/autoload/unite/custom.vim
235.000  002.131  001.196: sourcing /home/david/.vim/plugged/unite.vim/plugin/unite/bookmark.vim
235.507  000.444  000.444: sourcing /home/david/.vim/plugged/unite.vim/plugin/unite/buffer.vim
235.826  000.258  000.258: sourcing /home/david/.vim/plugged/unite.vim/plugin/unite/window.vim
236.443  000.562  000.562: sourcing /home/david/.vim/plugged/unite.vim/plugin/unite.vim
236.920  000.242  000.242: sourcing /home/david/.vim/plugged/vim-plugin-AnsiEsc/plugin/AnsiEscPlugin.vim
237.883  000.912  000.912: sourcing /home/david/.vim/plugged/vim-plugin-AnsiEsc/plugin/cecutil.vim
241.447  003.403  003.403: sourcing /home/david/.vim/plugged/tagbar/plugin/tagbar.vim
243.968  002.338  002.338: sourcing /home/david/.vim/plugged/vim-bookmarks/plugin/bookmark.vim
259.558  015.387  015.387: sourcing /home/david/.vim/plugged/vim-unimpaired/plugin/unimpaired.vim
259.838  000.172  000.172: sourcing /home/david/.vim/plugged/vim-autoread/plugin/autoread.vim
261.737  001.803  001.803: sourcing /home/david/.vim/plugged/vim-bufkill/plugin/bufkill.vim
262.409  000.114  000.114: sourcing /home/david/.vim/plugged/ultisnips/autoload/UltiSnips/map_keys.vim
262.570  000.719  000.605: sourcing /home/david/.vim/plugged/ultisnips/plugin/UltiSnips.vim
263.119  000.457  000.457: sourcing /home/david/.vim/plugged/QFEnter/plugin/QFEnter.vim
263.399  000.185  000.185: sourcing /home/david/.vim/plugged/vim-maximizer/plugin/maximizer.vim
263.553  000.060  000.060: sourcing /home/david/.vim/plugged/vim-litecorrect/plugin/litecorrect.vim
263.883  000.240  000.240: sourcing /home/david/.vim/plugged/vim-lexical/plugin/lexical.vim
265.936  001.939  001.939: sourcing /home/david/.vim/plugged/vim-textobj-quote/plugin/textobj/quote.vim
266.172  000.105  000.105: sourcing /home/david/.vim/plugged/vim-textobj-sentence/plugin/textobj/sentence.vim
266.587  000.317  000.317: sourcing /home/david/.vim/plugged/vim-wordy/plugin/wordy.vim
267.040  000.365  000.365: sourcing /home/david/.vim/plugged/vim-bufsurf/plugin/bufsurf.vim
267.503  000.020  000.020: sourcing /home/david/.vim/plugged/vim-misc/autoload/xolox/misc.vim
268.122  000.220  000.220: sourcing /home/david/.vim/plugged/vim-misc/autoload/xolox/misc/os.vim
269.151  000.142  000.142: sourcing /home/david/.vim/plugged/vim-misc/autoload/xolox/misc/cursorhold.vim
269.522  002.377  001.995: sourcing /home/david/.vim/plugged/vim-session/plugin/session.vim
269.865  000.105  000.105: sourcing /home/david/.vim/plugged/vim-misc/plugin/xolox/misc.vim
270.162  000.129  000.129: sourcing /home/david/.vim/plugged/rename.vim/plugin/rename.vim
270.930  000.600  000.600: sourcing /home/david/.vim/plugged/tabular/plugin/Tabular.vim
272.666  001.535  001.535: sourcing /home/david/.vim/plugged/FastFold/plugin/fastfold.vim
273.054  000.150  000.150: sourcing /home/david/.vim/plugged/limelight.vim/plugin/limelight.vim
273.296  000.069  000.069: sourcing /home/david/.vim/plugged/goyo.vim/plugin/goyo.vim
274.245  000.783  000.783: sourcing /home/david/.vim/plugged/vim-togglecursor/plugin/togglecursor.vim
274.720  000.286  000.286: sourcing /home/david/.vim/plugged/vim-livedown/plugin/livedown.vim
277.427  002.533  002.533: sourcing /home/david/.vim/plugged/fzf-preview.vim/plugin/fzf_preview.vim
278.932  000.653  000.653: sourcing /usr/local/share/nvim/runtime/plugin/gzip.vim
279.034  000.041  000.041: sourcing /usr/local/share/nvim/runtime/plugin/health.vim
279.317  000.230  000.230: sourcing /usr/local/share/nvim/runtime/plugin/man.vim
281.976  000.929  000.929: sourcing /usr/local/share/nvim/runtime/pack/dist/opt/matchit/plugin/matchit.vim
282.269  002.898  001.970: sourcing /usr/local/share/nvim/runtime/plugin/matchit.vim
282.904  000.579  000.579: sourcing /usr/local/share/nvim/runtime/plugin/matchparen.vim
284.506  001.546  001.546: sourcing /usr/local/share/nvim/runtime/plugin/netrwPlugin.vim
285.026  000.438  000.438: sourcing /usr/local/share/nvim/runtime/plugin/rplugin.vim
285.265  000.201  000.201: sourcing /usr/local/share/nvim/runtime/plugin/shada.vim
285.368  000.067  000.067: sourcing /usr/local/share/nvim/runtime/plugin/spellfile.vim
285.693  000.266  000.266: sourcing /usr/local/share/nvim/runtime/plugin/tarPlugin.vim
285.902  000.172  000.172: sourcing /usr/local/share/nvim/runtime/plugin/tohtml.vim
285.995  000.058  000.058: sourcing /usr/local/share/nvim/runtime/plugin/tutor.vim
286.337  000.305  000.305: sourcing /usr/local/share/nvim/runtime/plugin/zipPlugin.vim
289.587  013.595: loading plugins
289.852  000.264: loading packages
290.169  000.074  000.074: sourcing /home/david/.vim/plugged/vim-signature/after/plugin/signature.vim
290.496  000.185  000.185: sourcing /home/david/.vim/plugged/ultisnips/after/plugin/UltiSnips_after.vim
291.423  000.348  000.348: sourcing /home/david/.vim/plugged/tabular/autoload/tabular.vim
294.505  003.912  003.564: sourcing /home/david/.vim/plugged/tabular/after/plugin/TabularMaps.vim
295.134  001.111: loading after plugins
295.157  000.023: inits 3
298.264  003.107: reading ShaDa
302.983  000.920  000.920: sourcing /usr/local/share/nvim/runtime/scripts.vim
319.774  000.039  000.039: sourcing /usr/local/share/nvim/runtime/syntax/tex.vim
351.053  029.787  029.787: sourcing /usr/local/share/nvim/runtime/syntax/tex.vim
416.676  064.753  064.753: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/emoji.vim
432.332  127.347  032.767: sourcing /home/david/.vim/plugged/vimwiki/syntax/vimwiki.vim
433.661  000.400  000.400: sourcing /home/david/.vim/plugged/taskwiki/after/syntax/vimwiki.vim
437.389  000.315  000.315: sourcing /home/david/.vim/plugged/vimwiki/autoload/vimwiki/tags.vim
446.756  009.984  009.669: sourcing /home/david/.vim/plugged/vimwiki/ftplugin/vimwiki.vim
448.147  000.043  000.043: sourcing /usr/local/share/nvim/runtime/autoload/provider.vim
599.689  152.461  152.418: sourcing /home/david/.vim/plugged/taskwiki/ftplugin/vimwiki/taskwiki.vim
602.919  000.314  000.314: sourcing /home/david/.vim/plugged/taskwiki/after/ftplugin/vimwiki.vim
614.212  000.555  000.555: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions.vim
614.505  000.111  000.111: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/quickfix.vim
614.936  000.293  000.293: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline.vim
615.214  000.043  000.043: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/unite.vim
615.515  000.052  000.052: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/netrw.vim
615.784  000.055  000.055: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/fzf.vim
616.494  000.095  000.095: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/section.vim
617.103  000.312  000.312: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/highlighter.vim
617.619  001.449  001.041: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/term.vim
618.006  000.085  000.085: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/undotree.vim
618.340  000.086  000.086: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/tagbar.vim
618.675  000.059  000.059: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/bookmark.vim
619.275  000.334  000.334: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/branch.vim
619.614  000.082  000.082: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/fugitiveline.vim
619.943  000.075  000.075: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/syntastic.vim
635.869  000.107  000.107: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/nvimlsp.vim
636.338  000.192  000.192: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/whitespace.vim
636.904  000.104  000.104: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/po.vim
637.253  000.137  000.137: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline/extensions/wordcount.vim
637.720  000.264  000.264: sourcing /home/david/.vim/plugged/vim-airline/autoload/airline


```

**Comment by arooni** [comment]
on my mac, similar environment

```

![image](https://user-images.githubusercontent.com/556667/139601724-aa792606-dfba-4772-b459-68fb344a91ed.png)

```

**Comment by arooni** [comment]
Also have found that simply commenting out all my viewports on the ubuntu version and hitting save causes it to lock up for some reason?

**Comment by sdondley** [comment]
I think the newest version of taskwarrior, 2.6.1, has broken things badly. When opening a file with a single task in it, I get:

```
Error detected while processing function provider#python3#Call:
line   18:
Error invoking 'python_execute' on channel 3 (python3-script-host):
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/steve/git_repos/dotfiles2/nvim/plugged/taskwiki/taskwiki/errors.py", line 26, in wrapped_function
    original_function(*args, **kwargs)
  File "/Users/steve/git_repos/dotfiles2/nvim/plugged/taskwiki/taskwiki/decorators.py", line 10, in wrapped_function
    original_function(*args, **kwargs)
  File "/Users/steve/git_repos/dotfiles2/nvim/plugged/taskwiki/taskwiki/main.py", line 39, in update_from_tw
    c.load_tasks()
  File "/Users/steve/git_repos/dotfiles2/nvim/plugged/taskwiki/taskwiki/cache.py", line 237, in load_tasks
    for task in tasks:
  File "/Users/steve/Library/Python/3.9/lib/python/site-packages/tasklib/task.py", line 476, in __iter__
    self._result_cache = self._execute()
  File "/Users/steve/Library/Python/3.9/lib/python/site-packages/tasklib/task.py", line 508, in _execute
    return self.backend.filter_tasks(self.filter_obj)
  File "/Users/steve/Library/Python/3.9/lib/python/site-packages/tasklib/backends.py", line 334, in filter_tasks
    for line in self.execute_command(args):
  File "/Users/steve/Library/Python/3.9/lib/python/site-packages/tasklib/backends.py", line 297, in execute_command
    raise TaskWarriorException(error_msg)
tasklib.backends.TaskWarriorException: TASKRC override: /Users/steve/.taskrc
Configuration override rc.confirmation=no
Configuration override rc.dependency.confirmation=no
Configuration override rc.recurrence.confirmation=no
Configuration override rc.json.array=off
Configuration override rc.bulk=0
Configuration override rc.context=
Unable to find report that matches '11752b8e'.
Command used: task rc.confirmation=no rc.dependency.confirmation=no rc.recurrence.confirmation=no rc.json.array=off rc.bulk=0 rc.context= export 11752b8e
```

I'm on a mac using brew version of taskwarrior.

**Comment by sdondley** [comment]
This seemed to help: https://github.com/tools-life/taskwiki/issues/369

tldr: run `pip3 install -U tasklib`
